### PR TITLE
ROX-27350: Use trusted tasks

### PR DESCRIPTION
**Pull request by msugakov**
https://issues.redhat.com/browse/ROX-27480 is to follow up about moving fetch script.

### CODE REVIEW

File: image/scanner/rhel/konflux.Dockerfile [hunk 1]
```diff
@@ -16,19 +16,19 @@ ENV CI=1
 COPY . /src
 WORKDIR /src
 
-RUN scripts/konflux/fail-build-if-git-is-dirty.sh
+RUN .konflux/scripts/fail-build-if-git-is-dirty.sh
```
Comment (tommartensen) [inline image/scanner/rhel/konflux.Dockerfile:19]: We have this script in three different repositories. Can we run it as a separate task before `build-container`?

**Review by tommartensen — APPROVED** [review]
LGTM with one thought for the future.

File: image/scanner/rhel/konflux.Dockerfile (hunk 1 above)
Comment (msugakov) [reply image/scanner/rhel/konflux.Dockerfile:19]: Good idea to remove it from repos. Eventually this script only needed in `determine-image-tag` task.

I created https://issues.redhat.com/browse/ROX-27488

**Pull request merged**